python-frontend/src/test/java/org/sonar/python/lexer/PythonLexerTest.java

```diff
@@ -326,6 +324,18 @@ void fstring_incorrect_walrus_operator() {
       hasToken("}", PythonPunctuator.RCURLYBRACE),
       hasToken("'", PythonTokenType.FSTRING_END)));
   }
+
+  @Test
+  void fstring_walrus_looking_format_specifier() {
```

Comment:
I'm not sure this test was strictly needed, seeing it's quite similar to the one above it.

I think having tests ensuring the FP on rule S3457 is no longer raised (while the TP still is) might be more valuable: the underlying issue being already fixed and tested, we can just confirm that the rule works as intended and close the ticket.